test/fixtures/output/schema-registry/subject/update-help.golden

```diff
@@ -13,12 +13,17 @@ Update subject-level mode of subject "payments".
   $ confluent schema-registry subject update payments --mode readwrite
 
 Flags:
-      --compatibility string   Can be "backward", "backward_transitive", "forward", "forward_transitive", "full", "full_transitive", or "none".
-      --mode string            Can be "readwrite", "readonly", or "import".
-      --api-key string         API key.
-      --api-secret string      API key secret.
-      --context string         CLI context name.
-      --environment string     Environment ID.
+      --compatibility string         Can be "backward", "backward_transitive", "forward", "forward_transitive", "full", "full_transitive", or "none".
+      --compatibility-group string   The name for compatibility group.
```

Comment:
```suggestion
      --compatibility-group string   The name of the compatibility group.
```

Reply:
Can you define what a "compatibility group" is?

Reply:
In https://github.com/confluentinc/schema-registry/pull/2457 we enhanced the config API to have a few additional parameters. 
Eg. we specify the compatibility group for this subject as `application.version`. Then for each schema, you can define which compatibility group it belongs to by adding the following to its metadata
```
"application.version": 1
```
The compatibility check will be run for schemas that are in the same compatibility group.

Reply:
I wonder if "compatibility version" might make more sense here. By looking at this flag, I can't tell that it's asking for a version number.

Reply:
The `--compatibility-group` is specified to the property key, and the property value can be a number or string. Eg. you can have 
```
"application.version": "V1"
```
in metadata and set `--compatibility-group "application.version"`

Reply:
Got it, please make sure you add an example for this!